cookbook/recipes/database.rb

```diff
@@ -23,40 +23,44 @@
 # XXX: This configuration is currently for testing purposes only!
 ##
 
-include_recipe 'redisio::install'
+if node['redis']['local']
+  include_recipe 'redisio::install'
 
-unless node['redisio']['package_install']
-  include_recipe 'redisio::configure'
-  include_recipe 'redisio::enable'
+  unless node['redisio']['package_install']
+    include_recipe 'redisio::configure'
+    include_recipe 'redisio::enable'
+  end
 end
 
-mysql_service 'guardian' do
-  port '3306'
-  version '5.5'
-  initial_root_password 'change me'
-  action [:create, :start]
+if node['database']['local']
+  mysql_service 'guardian' do
+    port node['database']['port']
+    version '5.5'
+    initial_root_password node['database']['password']
+    action [:create, :start]
+  end
 end
 
 mysql2_chef_gem 'default' do
   action :install
 end
 
-mysql_database 'guardian' do
+mysql_database node['guardian']['database']['db_name'] do
```

Comment:
I'm not sure that we should use any of these resources for an RDS-hosted mysql instance. We should be able to configure the username and password and create the empty database in the `rds_instance` cloud-formation resource